### ISSUE TITLE
Pdf: When in zoom, next page should position to the top of the page automatically

### ISSUE THREAD

**Issue by andreimariusb** [koreader/kindlepdfviewer#831]
As the title says, I think it is better to have this functionality so I don't have to scroll to the top of the page at every page transition. It is distracting (I read technical pdfs).

If you are not able to do it (b/c of time) I will eventually fork the code and do it myself and if you like it you can integrate it into the main branch.

**Comment by tigran123** [comment]
I completely agree and, moreover, this is exactly how it works already. So, this is obviously a user error. This is normal as KPV is very complex software and 99% of users have absolutely no clue how to use it properly (and there is no documentation).

Ok, here is the "documentation", which I keep writing on two forums and here also, in answer to questions identical to yours:

There are two modes of using KPV: single-column mode (`Shift-S`) and two-column mode (`Shift-F`). Both modes require you to set a bbox prior to switching to each mode. To do this you press `A` first and then `Shift-X`. And then you go to the next page and do the same --- this ensures that bbox is set up for both odd and even pages. So, you have to do this for two consecutive pages. Then you (still in the `A` mode) go back to the page from which you wish to start reading and press `Shift-S` (for single-column) or `Shift-F` (for two-column). That is all there is to it. When you turn the pages it goes to the TOP of the page, exactly in the way that you desire.

No other zoom modes should be used as they are quite useless anyway (but they are there because some exotic books may require them).

**Comment by houqp** [comment]
Thanks @tigran123 for the explanation :) 

I added your document to wiki so next time you don't need to type it again ;p
https://github.com/koreader/kindlepdfviewer/wiki/Usage-Tips

**Comment by andreimariusb** [comment]
Thank you, now I understand what's with the bbox :+1: 

Is there a way to customize the distance the line moves so I can put it as close to the text as possible ? Since I read on the kindle 3 keyboard, I prefer to use all the screen space and when in full page mode ("A") the line moves with a predetermined increment so I either chop the text or I am little bit further than I could have been but it is not critical :D

It says "Note you can use the keyboard shortcut to select some position directly instead of keep moving cursor" - what shortcut ?

**Comment by chrox** [comment]
It should display ticks on both axis of the bbox ruler which reads "q w e r t y u i o p". You can press these shortcuts to jump to certain position directly.

**Comment by kai771** [comment]
@andreimariusb 

If you would like, you can try my fork of KPV called Librerator:
http://www.mobileread.com/forums/showthread.php?t=198742

It supports smaller and larger step for moving crosshairs that set BBox. (While setting BBox, pressing Q will select larger step, pressing Z would select smaller step, and pressing A will go back to default step).

Also, some shortcuts are changed: instead of Shift-S and Shift-F like in @tigran's post, you need to press only S or F.

**Comment by andreimariusb** [comment]
@kai771
Thanks. I will try it :+1:
